espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/w3c/models/ActiveInputSources.java

```diff
@@ -0,0 +1,33 @@
+package io.appium.espressoserver.lib.helpers.w3c.models;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.appium.espressoserver.lib.handlers.exceptions.InvalidArgumentException;
+
+/**
+ * Active Input Source defined in W3C spec
+ *
+ * (see https://www.w3.org/TR/webdriver/#terminology-0)
+ */
+public class ActiveInputSources {
+    private Map<String, InputSource> inputSources = new HashMap<>();
+
+    public void addInputSource(InputSource inputSource) throws InvalidArgumentException {
```

Comment:
InvalidArgumentException exception is unchecked

Reply:
It extends AppiumException which extends Exception

Was that changed in another PR?